### PR TITLE
webmaker.org goggles landing page updates

### LINKED ISSUE #161: Redirect users to new webmaker.org goggles landing page
Replace current goggles landing page by the one constructed in #1184 by redirecting to https://webmaker.org/goggles when someone hits https://goggles.webmaker.org/

(Judgement call for @Pomax: assumption is that it's easier to develop this inside of webmaker.org rather than goggles.webmaker.org)

### PR DESCRIPTION
**Pull request by secretrobotron**
Fixes #161
Fixes https://github.com/mozilla/goggles.webmaker.org/issues/182
Fixes https://github.com/mozilla/goggles.webmaker.org/issues/185

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -167,14 +167,11 @@ app.get('/healthcheck', function(req, res) {
   });
 });
 
+if (!env.get('WEBMAKER_REDIRECT_URL')) {
```
Comment (Pomax) [inline app.js]: let's just use this:
```
var landing = env.get("AUDIENCE") + "/goggles";
...
app.get("/", function(req, res) {
  res.redirect(302, landing);
});
```
no need for a new env. var in this case, we already know where webmaker.org lives

File: views/index.html [hunk 2]
```diff
@@ -1,172 +0,0 @@
...
-
-      <script src="{{ personaHost }}/include.js"></script>
-      <script src="/bower/webmaker-login-ux/dist/webmakerLogin.js"></script>
-    </div>
-
-    <div class="ui-wrapper ui-body-nobg">
-      <div class="banner">
-        <div class="header">
-          <div>
-            <img class="logo" src="img/goggles.png" alt="{{ gettext("X-Ray Goggles Logo") }}">
```
Comment (Pomax) [inline views/index.html]: warrants removing img/goggles.png?

File: views/index.html [hunk 3]
```diff
@@ -1,172 +0,0 @@
...
-        <div class="bookmarklet"><a href="" id="bookmarklet-link" class="ui-btn"><span>{{ gettext("Activate X-Ray Goggles") }}</span></a></div>
-      </div>
-      <div class="ui-bg-column">
-        <div class="ui-row">
-          <div class="g-4">
-            <h1>{{ gettext("See how Goggles work by swapping an image") }}</h1>
-            <ol>
-              <li>
-                <p>{{ gettext("Copy this image URL") }}<br>
-                <input type="text" autofocus value="{{ host }}/img/goggles-remixed.png"></p>
```
Comment (Pomax) [inline views/index.html]: removal of these images, too?

File: views/index.html [hunk 4]
```diff
@@ -1,172 +0,0 @@
...
-    {% endif %}
-  </head>
-  <body>
-    <div class="webmaker-outer-wrapper">
-      {% block header %}
-      <link rel="stylesheet" href="stylesheets/userbar-overrides.css">
-      <header id="webmaker-nav">
-        <nav class="webmaker-nav-container">
-          <div class="user-nav-container">
-            <a id="logo" href="{{ audience }}/{{localeInfo.lang}}/"><img src="/img/logo.png" alt="{{ gettext("Mozilla Webmaker") }}"></a>
```
Comment (Pomax) [inline views/index.html]: do we remove this image?

File: views/index.html [hunk 5]
```diff
@@ -1,172 +0,0 @@
...
-                <button class="btn btn-secondary-nav signin-button">{{ gettext("Sign in") }}</button>
-                <span class="logoutbutton hidden">{{ gettext("Sign out" )}}</span>
-              </li>
-            </ul>
-            <ul class="nav-list">
-              <li>
-                <a {% if page == 'index' %}class="on"{% endif %} id="nav-index" href="/{{localeInfo.lang}}">{{ gettext("Make") }}</a>
-                <div class="makenav-hover">
-                  <a href="{{ audience }}/{{localeInfo.lang}}/starter-makes" class="makenav-hover-item">
-                    <img src="/img/icon-make-template.png" alt="{{ gettext("From starter make") }}" />
```
Comment (Pomax) [inline views/index.html]: image

File: views/index.html [hunk 6]
```diff
@@ -1,172 +0,0 @@
...
-              </li>
-              <li>
-                <p>{{ gettext("activate the X-Ray Goggles by") }}</p>
-              </li>
-              <li>
-                <p>{{ gettext("With X-Ray Goggles now activated") }}</p>
-              </li>
-              <li>
-                <p>{{ gettext("Find the current X-Ray Goggles image") }}</p>
-                <p><img class="example" src="img/goggles.png" alt="{{ gettext("X-Ray Goggles Logo") }}"></p>
```
Comment (Pomax) [inline views/index.html]: image

File: views/index.html [hunk 7]
```diff
@@ -1,172 +0,0 @@
...
-
-            <h1>{{ gettext("Help") }}</h1>
-            <p>{{ gettext("If you need help") | safe }}</p>
-
-            <h1>{{ gettext("Remix any webpage!") }}</h1>
-            <p>{{ gettext("You can take X-Ray Goggles with you anywhere on the web:") }}</p>
-            <ol id="install">
-              <li>
-                <p>{{ gettext("Make sure your web browser's bookmarks bar is enabled.") }} </p>
-                <p class="screenshot step1" data-step="1"><img src="img/instructions/firefox1.png"></p>
```
Comment (Pomax) [inline views/index.html]: image - I think these are also dynamically replaced with chrome in some JS we no longer use now

File: views/index.html [hunk 8]
```diff
@@ -1,172 +0,0 @@
...
-              </li>
-              <li>
-                <p>{{ gettext("Drag the yellow button into your bookmarks") | safe }}</p>
-                <p class="screenshot step2" data-step="2"><img src="img/instructions/firefox2.png"></p>
```
Comment (Pomax) [inline views/index.html]: image - I think these are also dynamically replaced with chrome in some JS we no longer use now

File: app.js (hunk 1 above)
Comment (secretrobotron) [inline app.js]: @Pomax ah, is audience always pointing at a webmaker.org instance? (e.g. localhost:whatever)

File: views/index.html (hunk 4 above)
Comment (secretrobotron) [inline views/index.html]: Used in tests. I'll leave it for now.

File: views/index.html (hunk 3 above)
Comment (secretrobotron) [inline views/index.html]: beleted.

File: views/index.html (hunk 2 above)
Comment (secretrobotron) [inline views/index.html]: beleted.

File: views/index.html (hunk 6 above)
Comment (secretrobotron) [inline views/index.html]: beleted.

File: views/index.html (hunk 7 above)
Comment (secretrobotron) [inline views/index.html]: yeah. don't see image used anywhere else. beleted.

Comment (secretrobotron) [inline views/index.html]: Actually, `/public/js/browser-screen.js` uses them. I'll leave them there for now.

File: views/index.html (hunk 8 above)
Comment (secretrobotron) [inline views/index.html]: Same as above.

**Comment by secretrobotron** [comment]
@gesa @Pomax R?

**Comment by gesa** [comment]
R+

**Pull request merged**